File: Algorithm/javascript/BAEKJOON/CLASS 2/2751.js
// TRY 2
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdput,
});

let input = [];
rl.on("line", (line) => {
  if (!line) {
    rl.close();
  } else {
    input.push(line);
  }
}).on("close", () => {
  solution(input);
  process.exit();
});
function solution(input) {
  let [N, ...numbers] = input.map((e) => Number(e));
  let result = numbers.sort((a, b) => a - b);
  console.log(result.join("\n"));
}

// // TRY 1
// const readline = require("readline");
// const rl = readline.createInterface({
//   input: process.stdin,
//   output: process.stdput,
// });

// let input = [];
// rl.on("line", (line) => {
//   if (!line) {
//     rl.close();
//   } else {
//     input.push(line);
//   }
// }).on("close", () => {
//   solution(input);
//   process.exit();
// });

// function solution(input) {
//   let N = Number(input[0]);
//   let numbers = [];
//   for (let i = 1; i <= N; i++) {
//     numbers.push(i);
//   }

//   let sortedNumbers = numbers.sort();

//   for (number of sortedNumbers) {
//     console.log(number);
//   }
// }

File: Algorithm/python/BAEKJOON/CLASS 2/2751.py
# TRY 4
import sys

N = int(input())
numbers = []
for _ in range(N):
    numbers.append(int(sys.stdin.readline()))

for i in range(N-1, 0, -1):
    for j in range(i):
        if numbers[j] > numbers[j+1]:
            numbers[j] ,numbers[j+1] = numbers[j+1], numbers[j]

for number in numbers:
    print(number)

# # TRY 3
# import sys
#
# N = int(input())
# numbers = []
# for _ in range(N):
#     numbers.append(int(sys.stdin.readline()))
#
# for number in sorted(numbers):
#     print(number)

# # TRY 2
# N = int(input())
# numbers = []
# for _ in range(N):
#     numbers.append(int(input()))
#
# for i in range(N-1, 0, -1):
#     for j in range(i):
#         if numbers[j] > numbers[j+1]:
#             numbers[j] ,numbers[j+1] = numbers[j+1], numbers[j]
#
# for number in numbers:
#     print(number)

# # TRY 1
# N = int(input())
# numbers = []
# for _ in range(N):
#     numbers.append(int(input()))
#
# for i in range(N-1, -1, -1):
#     for j in range(i):
#         temp1 = numbers[j]
#         temp2 = numbers[j+1]
#         if numbers[j] > numbers[j+1]:
#             numbers[j] = temp2
#             numbers[j+1] = temp1
# for number in numbers:
#     print(number, end="\n")